scss/civicrm/administer/settings/_sms-provider.scss

```diff
@@ -0,0 +1,14 @@
+&.page-civicrm-admin-sms-provider {
+  .crm-container {
+    .crm-content-block {
+      overflow: hidden;
```

Comment:
why only here? why the exception?

Reply:
Generally we have buttons on top and bottom on admin pages so, it takes radius of surrounding elements. In this case table is not taking border-radius so, we are forcing to hide and show rounded corners by doing overflow: hidden